### PR TITLE
Add Alpine 3.10 to build matrix

### PR DESCRIPTION
**Pull request by frankkoornstra**
# Usabilla PHP Docker Template

Reviewers: @fabiotc @rdohms @WyriHaximus @renatomefi @frankkoornstra @agustingomes @cvmiert

## Type

Please specify the type of changes being proposed:

| Q                 | A
| ----------------- | -----
| Documentation?    | ~yes~/no
| Dockerfile change?| ~yes~/no
| Build feature?    | yes/~no~
| Apply CVE Patch?  | ~yes~/no
| Remove CVE Patch? | ~yes~/no

## Changelog

- Added [Alpine 3.10](https://alpinelinux.org/posts/Alpine-3.10.0-released.html) to the build matrix for PHP 7.2 & 7.3

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,16 +19,20 @@ build-cli: clean-tags
 	./build-php.sh cli 7.2 3.7
 	./build-php.sh cli 7.2 3.8
 	./build-php.sh cli 7.2 3.9
+	./build-php.sh cli 7.2 3.10
```
Comment (WyriHaximus) [inline Makefile]: There is no `3.10` tag available upstream: https://hub.docker.com/_/php

Comment (cvmiert) [reply Makefile]: I suppose we'll just have to wait for it then.

**Comment by WyriHaximus** [comment]
Just restarted the build since `3.10` tags are now available: https://hub.docker.com/_/php?tab=tags

**Comment by WyriHaximus** [comment]
Just restarted the build since `3.10` tags are now available: https://hub.docker.com/_/php?tab=tags

**Comment by WyriHaximus** [comment]
:tada: !

File: Makefile (hunk 1 above)
Comment (WyriHaximus) [reply Makefile]: It is now, this is passing and merged :tada:

**Pull request merged**